lib/hammer_cli/subcommand.rb

```diff
@@ -11,6 +11,7 @@ def initialize(names, description, subcommand_class, options = {})
         @subcommand_class = subcommand_class
         @hidden = options[:hidden]
         @warning = options[:warning]
+        super(@names, @description, @subcommand_class)
```

Comment:
If you call this, I wonder if names, description and subclass_command aren't already stored. I think you can drop `@names =` etc.

Reply:
Not really, AFAIR I've added this line because something wasn't initialized in clamp itself, crashing the commands.

Also, I rather call `super` with explicit arguments because implicitly it would send `params` as well, which is not supported by clamp.